File: ID007/dart/program.dart
import 'dart:io';

void main(List<String> args) {
  var a = stdin.readLineSync()?.split(" ").map(int.parse).toList() ?? [];
  var cnt = 0;
  for (var i = 1; i <= a[0]; i++) {
    if (i % a[1] == 0 || i % a[2] == 0) {
      cnt++;
    }
  }
  print(cnt);
}

File: ID007/fsharp/Program.fs
let [n; x; y] = stdin.ReadLine().Split(" ") |> Array.toList |> List.map int
let mutable cnt = 0
for i = 1 to n do
    if i % x = 0 || i % y = 0 then cnt <- cnt + 1
printfn "%d" cnt